docs/getting-started.mdx

```diff
@@ -5,38 +5,152 @@ description: Quick start guide for Xata's PostgreSQL platform
+## 1. Sign up
 
-1. **Install the CLI**
-   ```bash
-   curl -fsSL https://xata.io/install.sh | bash
-   ```
+Create your free Xata account at [console.xata.io](https://console.xata.io). You can sign up with GitHub, Google, or your email address.
 
-2. **Authenticate with Xata**
-   ```bash
-   xata auth login
-   ```
+![Xata sign-up page](assets/images/getting-started/signup.png)
 
-3. **Initialize your project**
-   ```bash
-   xata init
-   ```
+## 2. Create a project
 
-## Next steps
+After signing in, create a new project. A project is the top-level container for all your branches and Postgres instances.
 
-There are two primary workflows Xata supports natively.
+![Create project page](assets/images/getting-started/create-project.png)
 
-- **Developer workflow**: Set up staging and development environments with our [developer workflow](/tutorials/developer-workflow)
-- **Production migration**: Move your existing PostgreSQL database to Xata by [migrating to Xata](/tutorials/migrate-to-xata)
+## 3. Create a branch
 
-## Additional resources
+Within your project, create your first branch (e.g., `main`). A branch is a Postgres cluster—a collection of one or more Postgres instances, which can include a primary and optional replicas. When creating your main branch, you can choose the Postgres version, region, and instance size to fit your needs. This branch can serve as your production branch or a staging environment. You can branch off `main` at any time to create isolated development or test environments.
 
-Want to dive in further? Check out these pages.
+![Create branch page](assets/images/getting-started/create-branch.png)
 
-- [Architecture overview](/architecture) - Deep dive into Xata's technical architecture
-- [Core concepts](/core-concepts) - Learn about branching, staging, and data anonymization
-- [Command line interface](/cli) - Complete CLI reference
-- [GitHub actions](/github-actions) - Automate your development workflow
+## 4. Add sample xata (data)
+
+Navigate to the **Queries** section in your project and run the following SQL to create sample tables and data:
+
+```sql
+CREATE TABLE products(id SERIAL PRIMARY KEY,name TEXT,price NUMERIC(7,2));
+CREATE TABLE orders(id SERIAL PRIMARY KEY,created_at TIMESTAMPTZ DEFAULT now());
+CREATE TABLE order_items(order_id INT REFERENCES orders(id),product_id INT REFERENCES products(id),qty INT,PRIMARY KEY(order_id,product_id));
+INSERT INTO products(name,price) SELECT LEFT(md5(i::text),8),(random()*90+10)::numeric(7,2) FROM generate_series(1,10)i;
+WITH o AS (INSERT INTO orders DEFAULT VALUES RETURNING id) INSERT INTO order_items(order_id,product_id,qty) SELECT o.id,pid,(1+floor(random()*3))::int FROM o,(SELECT id pid FROM products ORDER BY random() LIMIT 5)p;
+```
+
+![Queries/SQL editor page](assets/images/getting-started/queries.png)
+
+## 5. View your branch schema and enable schema history
```

Comment:
Will we have a separate page for installing the Xata cli? If not, I would separate the installation of the CLI in it's own step, so we can link to it.

Reply:
We have reference docs, but I'll separate.